packages/esm-amd-loader/package.json

```diff
@@ -4,6 +4,7 @@
   "description": "Minimal AMD-style loader for replicating ES module behavior.",
```

Comment:
Let's add /test/ to esm-amd-loader/.npmignore to avoid any issues with having multiple nested package.json files in our published sources for this repo.

Reply:
It's already there.